tests/test_cases/test_cocotb/test_cocotb.py

```diff
@@ -1235,5 +1235,20 @@ def test_assertion_is_failure(dut):
     assert False
 
 
+class MyException(Exception):
+    pass
+
+@cocotb.test(expect_error=MyException)
+def test_expect_particular_exception(dut):
+    yield Timer(1)
+    raise MyException()
+
+
+@cocotb.test(expect_error=(MyException, AssertionError))
+def test_expect_exception_list(dut):
```

Comment:
Adding AssertionError here is a bit weird - I don't think that expect_error can handle that today?

Reply:
Hmmm, not sure. I just added something to ensure a tuple works, I'll change to `ValueError`.